scrapers/koji_scraper.py

```diff
@@ -0,0 +1,151 @@
+# SPDX-License-Identifier: GPL-3.0+
+from scrapers.base import BaseScraper
+from purview.models.koji import KojiBuild, KojiTask
+from purview.models.user import User
+import purview.utils.general as utils
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class KojiScraper(BaseScraper):
+
+    def run(self, since=None):
+        log.info('Starting initial load for Koji')
+        # Initialize a start date from which all builds must be fetched
+        # If no input is given by the user, fetch builds from the past two years
+        if since is None:
+            since = self.default_since
+        else:
+            since = utils.timestamp_to_datetime(since)
+
+        self.builds = self.get_koji_builds(since)
+        self.update_neo4j(self.builds)
+        log.info('Initial load of Koji builds complete!')
+
+    def get_koji_builds(self, start_date):
+        # SQL query to fetch all builds from start date until now
+        log.info('Getting all Koji builds since {0}'.format(start_date))
+        sql_query = """
+            SELECT build.*, events.time as creation_time, package.name as package_name
+            FROM build
+            LEFT JOIN events ON build.create_event = events.id
+            LEFT JOIN package ON build.pkg_id = package.id
+            WHERE events.time IS NOT NULL AND events.time >= '{}'
+            """.format(start_date)
+
+        log.debug("Query:", sql_query)
+        builds = self.teiid.query(sql=sql_query, retry=3)
+        log.info('Successfully fetched {0} builds from teiid'.format(len(builds)))
+        return builds
+
+    def get_task(self, task_id):
+        # SQL query to fetch task related to a certain build
+        sql_query = """
+            SELECT *
+            FROM brew.task
+            WHERE id = {}
+            """.format(task_id)
+
+        # Getting tasks for the builds from teiid
```

Comment:
This comment should be more generic. You are just getting a task. It doesn't have to be a build task.